include/tables/XAttrTable.h

```diff
@@ -124,6 +124,8 @@ struct XAttrRow {
     opWriter.EndObject();
 
     opWriter.EndObject();
+
+    LOG_INFO("script:" << sbOp.GetString());
```

Comment:
remove the log or if you need to keep it just write a more meaningful one

Reply:
fixed - it was leftover debug code